com.sap.cloud.lm.sl.mta/src/test/java/com/sap/cloud/lm/sl/mta/resolvers/v2_0/DescriptorPlaceholderResolverTest.java

```diff
@@ -23,13 +23,13 @@
 @RunWith(value = Parameterized.class)
 public class DescriptorPlaceholderResolverTest {
 
-    private final String deploymentDescriptorLocation;
-    private final String platformLocation;
-    private final String targetLocation;
+    protected final String deploymentDescriptorLocation;
+    protected final String platformLocation;
+    protected final String targetLocation;
```

Comment:
targetLocation is always the same => extract it as constant so it should not be passed as input for every test.
Check all versions of DescriptorValidatorTest.